### ISSUE TITLE
October 2024 Open Data - distribute

### ISSUE THREAD

**Issue by damonmcc** [NYCPlanning/data-engineering#1279]
> The following items are available for Socrata publication: 
> Zoning
> ZTLDB
> MIH
> EDes
> ZAP
> POPS
> FacDB
> LION Districts
> 
> @alexrichey @damonmcc @fvankrieken @sf-dcp 
> fyi @jackrosacker @croswell81  

 _Originally posted by @caseysmithpgh in [#1083](https://github.com/NYCPlanning/edm-overview/issues/1083#issuecomment-2498091411)_

**Comment by damonmcc** [comment]
@alexrichey you mentioned yesterday starting to distribute Open Data datasets. does that list in the description seem accurate?

**Comment by alexrichey** [comment]
@damonmcc Yes, distributing all of those, and will comment if any in particular aren't going out.

**Comment by fvankrieken** [comment]
<img width="213" alt="Image" src="https://github.com/user-attachments/assets/fe6f327f-c69e-4a85-9ca1-8fa007f4f5e1" />

**Comment by damonmcc** [comment]
did ZAP and EDes datasets. updated the Open Data Version column in the [Open Data excel](https://nyco365.sharepoint.com/:x:/r/sites/NYCPLANNING/itd/edm/_layouts/15/Doc.aspx?sourcedoc=%7B6735537b-8d8e-437a-b5e5-3a6b54194b24%7D&action=edit&wdenableroaming=1&wdlcid=en-US&wdorigin=Teams-HL.Sharing.DirectLink.Copy.LOF&wdhostclicktime=1734629518613&wdredirectionreason=Force_SingleStepBoot&wdinitialsession=2a2b33b6-e1b0-d655-31ed-6ebe4e2c927f&wdrldsc=2&wdrldc=1&wdrldr=ServiceCreationTimedOut)

**Comment by sf-dcp** [comment]
I did 2 datasets for zoning. Not sure if there are more to check "zoning" off the list?
